### PR TITLE
[skip-ci] Add RNTuple release notes

### PR DESCRIPTION
**Pull request by enirolf**

### CODE REVIEW

**Comment by dpiparo** [comment]
Excellent notes. Do we want to include brief mentions of the PoW items which were addressed (clearly w/o linking to the PoW)

File: README/ReleaseNotes/v632/index.md [hunk 1]
```diff
@@ -71,6 +71,19 @@ TChain::Add("/path/to/tree/subdir[0-9]/*.root")
...
+## RNTuple
+ROOT's experimental successor of TTree has seen a number of updates since the last release. Specifically, 6.32 includes the following changes:
+
+- A major refactoring of the interface, improving consistency across different parts and improving overall robustness. **Note that this is a breaking change with regard to 6.30!**
+- Support has been added for several new field types: `std::unordered_set<T>`, `std::map<K,V>`, `std::unordered_map<K,V>`
+- A new `RNTupleInspector` utility class has been added, to provide information about the on-disk metadata of an RNTuple.
+- A new `RNTupleParallelWriter` class has been added, providing (initial) support for parallel writing of RNTuples.
+- A new static method `RFieldBase::Check()` has been added, which produces a support status report of a type with regards to RNTuple I/O.
+- A new internal `RNTupleMerger` class has been added, enabling the merging of different page sources into one page sink.
+- Many bug fixes and improvements.
```
Comment (vepadulano) [inline README/ReleaseNotes/v632/index.md:83]: Small addition:
- It is possible to use the `RNTupleView` with an external address with type erasure, e.g. `std::shared_ptr<void> data{new float()}; auto view = reader->GetView("pt", data);`. This enables use cases such as reading one specific entry of one specific field into a previously allocated memory location.

**Review by vepadulano — APPROVED** [review]

File: README/ReleaseNotes/v632/index.md [hunk 2]
```diff
@@ -71,6 +71,26 @@ TChain::Add("/path/to/tree/subdir[0-9]/*.root")
...
+## RNTuple
+ROOT's experimental successor of TTree has seen a number of updates since the last release. Specifically, 6.32 includes the following changes:
+
+- A major refactoring of the interface, improving consistency across different parts and improving overall robustness. **Note that this is a breaking change with regard to 6.30!**
+- Support has been added for several new field types: `std::unordered_set<T>`, `std::map<K,V>`, `std::unordered_map<K,V>`
+- Support has been added for on-disk half-precision (IEEE 754-2008 16-bit) float fields. This can be enabled through `RField<float>::SetHalfPrecision()`. On reading, values of such fields are represented as regular, 32-bit floats.
+- A new `RNTupleInspector` utility class has been added, to provide information about the on-disk metadata of an RNTuple.
+- A new `RNTupleParallelWriter` class has been added, providing (initial) support for parallel writing of RNTuples.
+- A new static method `RFieldBase::Check()` has been added, which produces a support status report of a type with regards to RNTuple I/O.
+- A new internal `RNTupleMerger` class has been added, enabling the merging of different page sources into one page sink.
```
Comment (jblomer) [inline README/ReleaseNotes/v632/index.md:83]: Perhaps we can mention here that RNTuple data can thus be merged with `hadd`

**Comment by jblomer** [comment]
> Excellent notes. Do we want to include brief mentions of the PoW items which were addressed (clearly w/o linking to the PoW)

We should mention the zero-copy bulk read option

File: README/ReleaseNotes/v632/index.md [hunk 3]
```diff
@@ -71,6 +71,26 @@ TChain::Add("/path/to/tree/subdir[0-9]/*.root")
 ```
 This grabs all the root files in subdirectories that have a name starting with `subdir` and ending with some digit.
 
+## RNTuple
+ROOT's experimental successor of TTree has seen a number of updates since the last release. Specifically, 6.32 includes the following changes:
+
+- A major refactoring of the interface, improving consistency across different parts and improving overall robustness. **Note that this is a breaking change with regard to 6.30!**
```
Comment (jblomer) [inline README/ReleaseNotes/v632/index.md:77]: Perhaps let's also mention the RC2 on-disk format

**Comment by jblomer** [comment]
Here or in the RDataFrame section: support for RNTuple chains, improved multi-threaded work scheduling

File: README/ReleaseNotes/v632/index.md [hunk 4]
```diff
@@ -82,6 +82,29 @@ index but just does a copy in memory. Notably, this improves processing efficien
...
+- It is now possible to use the `RNTupleView` with an external address with type erasure, e.g.:
+  ```cpp
+  std::shared_ptr<void> data{new float()};
+  auto view = reader->GetView("pt", data);
+  ```
+  This enables use cases such as reading one specific entry of one specific field into a previously allocated memory location.
+- Further integration with [RDataFrame](#rdataframe): it is now possible to create RDataFrame for chains of RNTuples. This addition also comes with improvements to the multi-threaded work scheduling.
+- Many additional bug fixes and improvements.
+
+Please, report any issues regarding the above mentioned features should you encounter them. RNTuple is still experimental and is scheduled to become production grade by end of 2024. Thus, we appreciate feedback and suggestions for improvement.
```
Comment (jblomer) [inline README/ReleaseNotes/v632/index.md:106]:
```suggestion
Please, report any issues regarding the above mentioned features should you encounter them. RNTuple is still experimental. The on-disk format is scheduled to be finalized by the end of 2024. Thus, we appreciate feedback and suggestions for improvement.
```

**Review by jblomer — APPROVED** [review]

**Pull request merged**